admin/lib/errors/pupil.js

```diff
@@ -17,9 +17,9 @@ module.exports.addPupil = {
   'dobNoFuture': 'Date of birth can\'t be in the future',
   'dobRequired': 'Date of birth can\'t be blank',
   'dobInvalidChars': 'Entry must be a number',
-  'dob-day': 'Please check “Day”',
-  'dob-month': 'Please check “Month”',
-  'dob-year': 'Please check “Year”',
+  'dob-day': 'Please check \\“Day\\”',
```

Comment:
I suggest we don't need to escape the quotes by default - the string is fine.  If you want to write them into csv data then the csv package should be responsible for escaping where needed.

Reply:
@jon-shipley any suggestion on how to escape double quotes within single quotes w/o escape chars?
That was my original intention but it doesn't seem like it's possible.

Reply:
Why do you need to escape them?  The error message string is fine without the escapes.